### ISSUE TITLE
Autorepeat gets stuck when pressing teo arrows

### ISSUE THREAD

**Issue by rsola** [termux/termux-app#2156]
**Problem description**

**Steps to reproduce**

**Expected behavior**

**Additional information**

* Termux application version:
* Android OS version:
* Device model:

**Comment by suhan-paradkar** [comment]
I can confirm this on latest debug artifact....
But strangely... It stops after opening the context menu...

https://user-images.githubusercontent.com/76830230/124353514-93b55500-dc24-11eb-8360-2af636466152.mp4

**Comment by rsola** [comment]
Yes, sorry, I had accidentally pressed Submit before I wrote the full text.

**Comment by agnostic-apollo** [comment]
Confirmed. AutoRepeat is implemented [here](https://github.com/termux/termux-app/blob/v0.116/app/src/main/java/com/termux/app/terminal/io/extrakeys/ExtraKeysView.java#L306). Will look into it. Other stuff needs to be done for extra keys too, so might do it with it at some point.

**Comment by agnostic-apollo** [comment]
It's likely because previous executioner should be shutdown before starting a new one and losing reference to the old one. Just a guess. But if there is a need to press multiple keys at the same time, which likely there is, then will have to create a list of executioners.
